worlds/civ_6/docs/setup_en.md

```diff
@@ -0,0 +1,51 @@
+# Setup Guide for Civilization VI Archipelago
+
+This guide is meant to help you get up and running with Civlization VI in your Archipelago run. Note that this requires you to have both Rise & Fall as well as Gathering Storm installed. This will not work unless both of those DLCs are enabled.
+
+## Requirements
+
+The following are required in order to play Civ VI in Archipelago
+
+- Windows OS (Firaxis does not support the necessary tooling for Mac, Linux is yet to bet verified)
+
+- Installed [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases) v0.4.5 or higher.\
+   **Make sure to install the Generator if you intend to generate multiworlds.**

```

Comment:
```suggestion
- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases) v0.4.5 or higher.
```
The install generator option got removed a few versions back, before 0.4.5 I think -- it always installs now